### PR TITLE
test(AI&Alerts): Demonstrating a PR

### PR DESCRIPTION
**Pull request by paperclypse**
This is for training purposes. DO NOT MERGE.

### CODE REVIEW

File: src/content/docs/alerts-applied-intelligence/transition-guide.mdx [hunk 1]
```diff
@@ -10,6 +10,8 @@ redirects:
   - /docs/alerts-applied-intelligence/new-relic-alerts/learn-alerts/alerts-ai-transition-guide-2022 
 ---
 
+TEST
```
Comment (paperclypse) [inline src/content/docs/alerts-applied-intelligence/transition-guide.mdx:13]: Hey, I don't think we want to all caps TEST. That seems ridiculous

File: src/content/docs/alerts-applied-intelligence/transition-guide.mdx [hunk 2]
```diff
@@ -10,6 +10,8 @@ redirects:
...
+
 Over the course of several months during 2022, new functionalities in New Relic One will replace alerting functionalities reaching end of life (EOL). We are providing these early notices so you can plan accordingly and make a smooth transition.
```
Comment (paperclypse) [inline src/content/docs/alerts-applied-intelligence/transition-guide.mdx:16]: We should really have put the TEST here

Comment (paperclypse) [reply src/content/docs/alerts-applied-intelligence/transition-guide.mdx:16]: I can add comments to my comment.

File: src/content/docs/alerts-applied-intelligence/transition-guide.mdx (hunk 1 above)
Comment (paperclypse) [inline src/content/docs/alerts-applied-intelligence/transition-guide.mdx:13]:
```suggestion
Test
```

**Review by akristen — APPROVED** [review]

**Pull request closed without merging**